python/src/uagents/resolver.py

```diff
@@ -4,11 +4,38 @@
 from typing import Dict, List, Optional, Tuple
 import random
 
-from uagents.config import DEFAULT_MAX_ENDPOINTS
+from uagents.config import (
+    DEFAULT_MAX_ENDPOINTS,
+    TESTNET_PREFIX,
+    MAINNET_PREFIX,
+    AGENT_PREFIX,
+)
 from uagents.network import get_almanac_contract, get_name_service_contract
 
+# pylint: disable=arguments-differ
```

Comment:
I think this will not be needed if the update from my comment below is implemented. In general these pylint rules are there for good reasons and we should have a better reason to ignore them, which is sometimes the case, but I'm not convinced it is here.